src/dlgIRC.cpp

```diff
@@ -173,7 +173,7 @@ QPair<bool, QString> dlgIRC::sendMsg(const QString& target, const QString& messa
         mPingStarted = QDateTime::currentMSecsSinceEpoch();
     }
 
-    bool rv = connection->sendCommand(command);
+    connection->sendCommand(command);
```

Comment:
If we ever make use of Communi's command filter this -can- return `false`.  Since we don't and this return value doesn't indicate success or failure to send it isn't very useful at this point.